synfig-studio/src/synfigapp/actions/waypointsetsmart.cpp

```diff
@@ -452,3 +469,21 @@ Action::WaypointSetSmart::prepare()
 
 	throw Error(_("Unable to determine how to proceed. This is a bug."));
 }
+void
+Action::WaypointSetSmart::redirect_value_desc_set_action()
```

Comment:
rename to `redirect_to_value_desc_set_action` (or `redirect_to_action_value_desc_set`?)

Reply:
ok, makes sense.